tests/decorators/test_clickjacking.py

```diff
@@ -1,45 +1,47 @@
-from unittest import TestCase
-
 from django.http import HttpRequest, HttpResponse
 from django.middleware.clickjacking import XFrameOptionsMiddleware
+from django.test import SimpleTestCase
 from django.views.decorators.clickjacking import (
     xframe_options_deny,
     xframe_options_exempt,
     xframe_options_sameorigin,
 )
 
 
-class XFrameOptionsDecoratorsTests(TestCase):
+class XFrameOptionsDenyTests(SimpleTestCase):
```

Comment:
This has been changed from `TestCase` to `SimpleTestCase` for two reasons:

1. `TestCase` doesn't seem to await async functions properly. I'm not sure exactly why, but this doesn't seem like the right PR to fix it in.
2. As per the [Django docs](https://docs.djangoproject.com/en/dev/topics/testing/tools/#testcase) since we're not using a database for any of these tests, we should be using `SimpleTestCase` instead of `TestCase`.